packages/shared/src/components/sidebar/DiscoverSection.tsx

```diff
@@ -61,19 +62,16 @@ export function DiscoverSection({
         </span>
       ),
     },
-  ];
-
-  if (searchValue === SearchExperiment.Control) {
-    discoverMenuItems.push({
+    {
       icon: (active: boolean) => (
         <ListIcon Icon={() => <SearchIcon secondary={active} />} />
       ),
-      title: 'Search',
+      title: isControlSearch ? 'Search' : 'Post finder',
       hideOnMobile: true,
```

Comment:
We can remove this if not control search

Reply:
Oh, right. Yes, will make that based on experiment.

Reply:
Ah, right. Let me update it quickly.